sdk/Internal/StringUtil.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Sdk\Internal;
+
+class StringUtil
+{
+    public static function substr(string $value, int $offset, int $length): string
+    {
+        if (function_exists('mb_substr')) {
```

Comment:
Can you think of a particular place where `mb_substr` and `mb_strlen` wouldn't exist?  I can't, but I'm probably a bit jaded here.

Reply:
`mbstring` is a non-default extension (https://www.php.net/manual/en/mbstring.installation.php) while Docker image has it enabled by default.

This check in code could be replaced by the additional requirement in `composer.json`. But I'm not sure which is better. When the user doesn't have `mbstring` installed that means the user code doesn't care about Unicode at all. The library would not help.

Reply:
Thank you for the context.  I think this is sufficient for now; we can most certainly include that in `composer.json` if we ever decided we wanted to.  I like the direction you took.